caddyconfig/httpcaddyfile/builtins.go

```diff
@@ -181,6 +191,66 @@ func parseTLS(h Helper) ([]ConfigValue, error) {
 					cp.Curves = append(cp.Curves, h.Val())
 				}
 
+			case "clients":
+				ch := h.NewFromNextSegment() // new helper for client auth config
+				cp.ClientAuthentication = &caddytls.ClientAuthentication{}
+				for ch.Next() {
+					for ch.NextBlock(0) {
+						switch ch.Val() {
+						case "mode":
+							for ch.NextArg() {
+								cp.ClientAuthentication.Mode = ch.Val()
+							}
+						case "trusted_ca_certs":
```

Comment:
Let's rename this to `trusted_ca_cert` since it is most likely that the user will only specify one, and they can specify it multiple times too if they need to -- as opposed to the JSON field which is a slice.
```suggestion
						case "trusted_ca_cert":
```